CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## v1.2.0
+
+### Additions
+
+- Enabled definition of custom categories encoded as [JSON file(s)](https://olm.operatorframework.io/docs/tasks/validate-package/#validation) for the `operatorhub` validator. ([#4109](https://github.com/operator-framework/operator-sdk/pull/4109))
+
+### Changes
+
+- Improved `ansible-operator` and `helm-operator` help text. ([#4187](https://github.com/operator-framework/operator-sdk/pull/4187))
+- When generating bundles and packagemanifests, add all resources supported by OLM. ([#4137](https://github.com/operator-framework/operator-sdk/pull/4137))
+- `olm` and `run` subcommands will print aggregated resource errors when either OLM or an operator fail to install, respectively. ([#3787](https://github.com/operator-framework/operator-sdk/pull/3787))
+- Upgrade sigs.k8s.io/controller-runtime dependency version from `v0.6.2` to [`v0.6.3`](https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.6.3). ([#4062](https://github.com/operator-framework/operator-sdk/pull/4062))
+- For Golang based operators with multigroup support, fix scaffolded `controllers/<group>/suite_test.go` to have a correct `envtest.Environment.CRDDirectoryPaths` ([kubebuilder#1665](https://github.com/kubernetes-sigs/kubebuilder/pull/1665)). ([#4062](https://github.com/operator-framework/operator-sdk/pull/4062))
```

Comment:
This one should be in the bug fixes. Could you please move it before get the PR merge?